cmd/groundcover/root.go

```diff
@@ -49,6 +71,27 @@ groundcover, more data at: https://groundcover.com/docs`,
 	SilenceErrors: true,
 }
 
+func checkLatestVersionUpdate(ctx context.Context) (shouldUpdate bool, selfUpdater *selfupdate.SelfUpdater) {
```

Comment:
as a convention we don't use  named return values (although I like the feature true) so make the return statements explicit